src/main/resources/logback.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<configuration>
+    <appender name="stdout" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>[%d] %p %m \(%c:%t\)%n</pattern>
```

Comment:
I agree with you @gkleiman with not changing the format here. I still like the idea of the long format specification.